README.md

```diff
@@ -93,8 +93,8 @@ When using the _GITHUB_TOKEN_, the **minimum required permissions** are:
 | `releasedLabels`         | The [labels](https://help.github.com/articles/about-labels) to add to each issue and pull request resolved by the release. Set to `false` to not add any label. See [releasedLabels](#releasedlabels). | `['released<%= nextRelease.channel ? \` on @\${nextRelease.channel}\` : "" %>']-                                                                     |
 | `addReleases`            | Will add release links to the GitHub Release. Can be `false`, `"bottom"` or `"top"`. See [addReleases](#addReleases).                                                                                  | `false`                                                                                                                                              |
 | `draftRelease`           | A boolean indicating if a GitHub Draft Release should be created instead of publishing an actual GitHub Release.                                                                                       | `false`                                                                                                                                              |
-| `releaseNameTemplate`    | A [Lodash template](https://lodash.com/docs#template) to customize the github release's name                                                                                                           | `<%= nextverison.name %>`                                                                                                                            |
-| `releaseBodyTemplate`    | A [Lodash template](https://lodash.com/docs#template) to customize the github release's body                                                                                                           | `<%= nextverison.notes %>`                                                                                                                           |
+| `releaseNameTemplate`    | A [Lodash template](https://lodash.com/docs#template) to customize the github release's name                                                                                                           | `<%= nextRelease.version %>`                                                                                                                            |
```

Comment:
```suggestion
| `releaseNameTemplate`    | A [Lodash template](https://lodash.com/docs#template) to customize the github release's name                                                                                                           | `<%= nextRelease.name %>`                                                                                                                            |
```

i'm confused by your proposed change for this line since it doesnt seem to align with the implementation. should is still be `.name`?

Reply:
Sorry for the noise - I was confused as well as I faced a lot of issues. There is no nextversion object in the context only a nextVersion and the Version number itself is named version. The notes are still named notes.

See: https://semantic-release.gitbook.io/semantic-release/developer-guide/js-api#nextrelease